oneview_redfish_toolkit/services/manager_service.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Own libs
+from oneview_redfish_toolkit import multiple_oneview
+
+
+def get_current_manager():
+    map_resources = multiple_oneview.get_map_resources()
+    manager_uuid = list(map_resources.keys())[0]
```

Comment:
We discussed and @karolcatunda will change to get the OneView IP for the appliance uuid, and make a direct query on this OneView.